client/app/lib/components/contentModal/styl/contentModal.styl

```diff
@@ -275,11 +275,20 @@ main.main-container
     borderBox()
 
     .markdown-content
-      margin-top            10px
-      max-width             450px
-
-      pre code
-        overflow            auto
+      /* margin-top            10px */
+      /* max-width             450px */
+      padding                   10px 30px
+      
+      pre
+        margin                  0px
+        display                 inline-block
+        
+        code
+          overflow              auto
+          margin                0px
+          padding-top           20px
+          padding-bottom        20px
+          display               inline-block
```

Comment:
you probably need no styling for the `code` tag at all. But assuming you do again `display` defaults to `inline` and that should be fine.